source/install/cloud-changelog.md

```diff
@@ -4,12 +4,46 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-12-08](#release-2022-12-08)
 - [Release 2022-12-01](#release-2022-12-01)
 - [Release 2022-11-24](#release-2022-11-24)
 - [Release 2022-11-17](#release-2022-11-17)
 - [Release 2022-11-10](#release-2022-11-10)
 - [Release 2022-10-27](#release-2022-10-27)
-- [Release 2022-10-20](#release-2022-10-20)
+
+## Release 2022-12-08
+
+### Highlights
+
+#### Message Priority and Acknowledgments (Professional plan)
+ - Added message priority labels to the Threads view.
+ - Added support for enterprise users to request acknowledgements on posts and to acknowledge posts.
+
+#### Global Drafts
+ - Added a centralized Drafts view for draft messages.
+
+### Improvements
+
+#### User Interface (UI)
+ - Updated prepackaged version of plugins affected by React 17 upgrade.
+
+#### Administration
+ - The export file now contains the server version and a creation timestamp.
+ - The **System Console > Subscriptions** page now reflects whether the plan is monthly or annual.
+ - Added the ability to migrate from a monthly to an annual Cloud Professional plan in-product.
```

Comment:
I heard from Nick that these are not releasing yet, do I remove these from the changelog?

Reply:
Confirmed, feature flagged, not going out this release. Can be removed.